pipeline/vars/common.groovy

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env groovy
+/*
+    Common groovy methods that can be reused by the pipeline jobs.
+*/
+
+import groovy.json.JsonSlurper
+
+String composeId = ""
+String instanceName = ""
+
+def prepareNode() {
+    /*
+        Installs the required packages needed by the Jenkins node to
+        run and execute the cephci test suites.
+    */
+    sh '''
+        #!/bin/bash
+
+        echo "Initialize Node"
+        sudo yum install -y wget git-core python3
+
+        # Workaround: Disable IPv6 to have quicker downloads
+        sudo sysctl -w net.ipv6.conf.eth0.disable_ipv6=1
+
+        # Mount reesi for storing logs
+        if [ ! -d "/ceph" ]; then
+            echo "Mounting ressi004"
+            sudo mkdir -p /ceph
+            sudo mount -t nfs -o sec=sys,nfsvers=4.1 reesi004.ceph.redhat.com:/ /ceph
+        fi
+
+        # Copy the auth files from internal server
+        wget http://magna002.ceph.redhat.com/cephci-jenkins/.osp-cred-ci-2.yaml -O ${HOME}/osp-cred-ci-2.yaml
+        wget http://magna002.ceph.redhat.com/cephci-jenkins/.cephci.yaml -O ${HOME}/.cephci.yaml
+
+        # Install cephci pre-requisistes
+        rm -rf .venv
+        python3 -m venv .venv
+        source .venv/bin/activate
+        python -m pip install --upgrade pip
+        python -m pip install -r requirements.txt
+        deactivate
+    '''
+}
+
+def runTestSuite() {
+    /*
+        Execute the test suite by processing CI_MESSAGE for container
+        information along with the environment variables. The required
+        variables are
+            - sutVMConf
+            - sutConf
+            - testSuite
+            - addnArgs
+    */
+    println "Begin Test Suite execution"
+
+    // generate random instance name
+    def sec = sh(script: "echo \$(date +%s)", returnStdout: true).trim()
+    instanceName = "psi-${sec}"
```

Comment:
can we attach tier level string `tier0` into name string instead of `psi`

Reply:
this function looks like a common function, which can be used even for other tiers. so I think string `tier0` might not be correct here.